src/blueprints/tv2/helpers/tv2-show-style-blueprint-configuration-mapper.ts

```diff
@@ -19,10 +19,16 @@ interface CoreShowStyleBlueprintConfiguration {
   GfxSchemaTemplates: CoreGraphicsSchema[]
   DVEStyles: CoreSplitScreenConfiguration[]
   BreakerConfig: CoreBreaker[]
-  Transitions: { _id: string, Transition: string }[]
+  Transitions: Transition[]
   ShowstyleTransition: string
 }
 
+// SOF-2004
```

Comment:
What is the purpose of this comment? 
If it is to tell us which task this change is from, then that is in your commit message since we put our Jira Ids in those :)

Reply:
Ah, my bad. A lot of the software houses I worked in required me to comment on in association with what task ID something changed and who changed it. Since that isn't a thing here I will remove and not use that style of commenting.